visual-dotnet/SauceLabs.Visual/Utils/EnvVars.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace SauceLabs.Visual.Utils
+{
+    internal class EnvVars
+    {
+        internal static string? CustomId => Environment.GetEnvironmentVariable("SAUCE_VISUAL_CUSTOM_ID");
```

Comment:
it would be better from the constructor perspective if we keep string as not nullable value. What do you think about using ?? "" for null here to keep our types obvious? There is no performance cost at all with such a check